site/gatsby-site/src/components/index.js

```diff
@@ -2,3 +2,4 @@ export * from './theme';
 export mdxComponents from './mdxComponents';
 export Layout from './Layout';
 export Link from './Link';
+export MiddleLayout from './MiddleLayout';
```

Comment:
It looks like `middleLayout` is the filename and `MiddleLayout` is the exported symbol.